templates/faq/embedded-faq.html

```diff
@@ -0,0 +1,252 @@
+            {% endfor %}
+        </li>
+    {% endfor %}
+    </ul>
+
+    <h4 lang="en">
+        For more information please visit our web-site
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+    <h4 lang="ru">
+        Больше информации на нашем сайте
+        <a href="https://organicmaps.app/ru/">organicmaps.app</a>
+    </h4>
+    <h4 lang="de">
+        Für weitere Informationen besuchen Sie bitte unsere Website
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+    <h4 lang="es">
+        Para más información, visite nuestro sitio web
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+    <h4 lang="fr">
+        Pour plus d'informations, veuillez visiter notre site web
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+    <h4 lang="pt">
+        Para mais informações, visite o nosso sítio na web
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+    <h4 lang="pt-BR">
+        Para mais informações, visite o nosso sítio web
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+    <h4 lang="pl">
+        Aby uzyskać więcej informacji, odwiedź naszą stronę internetową
+        <a href="https://organicmaps.app/">organicmaps.app</a>.
+    </h4>
+    <h4 lang="tr">
+        Daha fazla bilgi için lütfen
+        <a href="https://organicmaps.app/tr/">organicmaps.app/tr/</a>
+        web sitemizi ziyaret edin.
+    </h4>
+    <h4 lang="uk">
+        Для отримання додаткової інформації, будь ласка, відвідайте наш веб-сайт
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+    <h4 lang="zh">
+        要获取更多信息，请访问我们的网站
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+
+</div>
+
+{% for faq_slug in faq_questions_en %}
+    {% set faq_page_en = get_page(path = "faq/" ~ faq_slug ~ "/index.md") %}
+    <dl id="{{ faq_slug | replace(from="/", to="-") }}">
+        <dt>
+            <h3 lang="en">{{ faq_page_en.title }}</h3>
+            {# Add translations #}
+            {% for lang in faq_languages %}
+                {% set translated_faq_page = faq_page_en.translations | filter(attribute="lang", value=lang) | first %}
+                <h3 lang="{{ lang | replace(from="-Hans", to="") }}">{{ translated_faq_page.title }}</h3>
+            {% endfor %}
+        </dt>
+
+        <dd lang="en">
+            {{ faq_page_en.content | safe }}
+        </dd>
+        {# Add translations #}
+        {% for lang in faq_languages %}
+            {% set translated_faq_page = faq_page_en.translations | filter(attribute="lang", value=lang) | first %}
+            {% set translated_faq_page = get_page(path=translated_faq_page.path) %}
+            <dd lang="{{ lang | replace(from="-Hans", to="") }}">
+                {{ translated_faq_page.content | safe }}
+            </dd>
+        {% endfor %}
+    </dl>
+{% endfor %}
+
+<a href="#" lang="en">Go to Top</a>
```

Comment:
Looks like adding these translations into config.toml would be more flexible.

Reply:
Added new translation key 'go-to-top'.